test/scripts/e2e-cp.sh

```diff
@@ -108,11 +108,11 @@ if [ "$OVN_GATEWAY_MODE" == "shared" ]; then
 fi
 
 if [ "$OVN_GATEWAY_MODE" == "local" ]; then
-  if [ "$SKIPPED_TESTS" != "" ]; then
-    SKIPPED_TESTS+="|"
-  fi
   # See https://github.com/ovn-org/ovn-kubernetes/labels/ci-ipv6 for details:
   if [ "$KIND_IPV6_SUPPORT" == true ]; then
+    if [ "$SKIPPED_TESTS" != "" ]; then
```

Comment:
if you put this here, then what about if its not v6 ? :D

Reply:
There is nothing to do on that case, since there is nothing to add.

Reply:
I mean `|` is needed if there are something previously and we are going to add something, in the case of local, ipv4 there is nothing to skip so we should not add it.

Reply:
ack so as of today there is no skip being done if its local and if its v4..

Reply:
right